app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -288,6 +290,24 @@ class TracingApi {
   }
 
   /**
+   * Centers the 3D view.
+   *
+   * @example
+   * api.tracing.centerTDView()
+   */
+  centerTDView = (): void => {
+    Store.dispatch(centerTDViewAction());
+  }
+
+  changeTDViewXY = (): void => changeTDView(OrthoViews.PLANE_XY);
```

Comment:
You're right, I just moved these methods from the camera controller, but the names are definitely not self-explanatory. These methods are used when clicking on the "xy"/... buttons in the 3D view. So, it only changes the 3D view.
How about `rotate3DViewToXY` etc. for the new names? What do you think?